Mergin/plugin.py

```diff
@@ -294,44 +295,12 @@ def project_status(self):
 
         try:
             pull_changes, push_changes, push_changes_summary = self.mc.project_status(self.path)
-            pull_added, pull_removed, pull_updated, pull_renamed = changes_from_metadata(pull_changes)
-            push_added, push_removed, push_updated, push_renamed = changes_from_metadata(push_changes)
-
-            pull_msg = "Pending changes from the latest server version: \n"
-            if pull_added:
-                pull_msg += f"added: {pull_added} \n"
-            if pull_removed:
-                pull_msg += f"removed: {pull_removed} \n"
-            if pull_updated:
-                pull_msg += f"updated: {pull_updated} \n"
-            if pull_renamed:
-                pull_msg += f"renamed: {pull_renamed} \n"
-
-            push_msg = "These local changes have been found: \n"
-            if push_added:
-                push_msg += f"added: {push_added} \n"
-            if push_removed:
-                push_msg += f"removed: {push_removed} \n"
-            if push_updated:
-                push_msg += f"updated: {push_updated} \n"
-            if push_renamed:
-                push_msg += f"renamed: {push_renamed} \n"
-
-            files_to_replace = ", ".join([file["path"] for file in push_changes["updated"] if "diff" not in file and ".gpkg" in file['path']])
-
-            msg = ''
-            if sum(len(v) for v in pull_changes.values()):
-                msg += pull_msg + "\n"
-            if sum(len(v) for v in push_changes.values()):
-                msg += push_msg
-                msg += pretty_summary(push_changes_summary)
-                msg += f"\nUnable to compare some of the modified local files with their server version - we will have to upload the whole file (history of the files will be lost): {files_to_replace}" if files_to_replace else ""
```

Comment:
This warning got lost - if a diffable file does not have diff info from geodiff, we should show a warning as a child item of such file